### ISSUE TITLE
time to update python-dateutil again#Help #Important

### ISSUE THREAD

**Issue by RinaWong** [boto/botocore#2353]
This is preventing me from installing pandas now, when installing pandas I am running into this and it's failing.
I tried to install an new version of python-dateutil before installing pandas but it still runs into this error:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
botocore 1.9.18 requires python-dateutil<2.7.0,>=2.1, but you have python-dateutil 2.7.3 which is incompatible.
```

Help!!!THX :(

**Comment by nateprewitt** [comment]
Hi @RinaWong, this version range was updated roughly in April of 2018 to allow any version of python-dateutil < 3.0.0. You should be free to use that dateutil version with any SDK released since April 23, 2018 (boto3 >= 1.7.7, awscli >= 1.15.7).

**Comment by RinaWong** [comment]
> Hi @RinaWong, this version range was updated roughly in April of 2018 to allow any version of python-dateutil < 3.0.0. You should be free to use that dateutil version with any SDK released since April 23, 2018 (boto3 >= 1.7.7, awscli >= 1.15.7).

Thx for your reply. My version is botocore == 1.9.18, python-dateutil == 2.7.3, but there is no awscli.
Then I pip install awscli==1.15.7, but there is runs into this error:
```
ERROR: Cannot uninstall 'colorama'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```
I'll keep trying......

**Comment by nateprewitt** [comment]
From the error it looks like you've installed a version of `colorama` through some other means. If you only need botocore and aren't using boto3 or the awscli, botocore 1.10.7 is the minimum version that will support that version of python-dateutil.

If you're having dependency conflicts, we'd recommend installing in a [virtualenv](https://virtualenv.pypa.io/en/latest/) to avoid these kinds of conflicts going forward.
